bom/pom.xml

```diff
@@ -149,6 +149,7 @@
         <groupId>io.netty</groupId>
         <artifactId>netty-common</artifactId>
         <version>${project.version}</version>
+      <!-- <version>4.1.94.Final-SNAPSHOT-CHERI</version> -->
```

Comment:
what is this for?

Reply:
Think this is a remnant from when I was starting to update everything to use the ported `netty-common` - but I'll delete for now